MapboxNavigation/NavigationViewController.swift

```diff
@@ -423,7 +423,11 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
     }
     
     func applyStyle() {
-        styles?.forEach { $0.apply() }
+        styles?.forEach {
```

Comment:
There’s no need for `forEach` here. Can you make this a normal `for` loop?

Reply:
it does handle the optional `styles` better, but shrug.

Reply:
The way I usually handle that is:

```swift
for style in styles ?? [] { … }
```